props/configs/src/main/resources/dh-defaults.prop

```diff
@@ -71,3 +71,47 @@ client.configuration.list=java.version,deephaven.version,barrage.version,groovy.
 # jar, and a class that is found in that jar. Any such keys will be made available to the client.configuration.list
 # as <key>.version.
 client.version.list=deephaven=io.deephaven.engine.table.Table,barrage=io.deephaven.barrage.flatbuf.BarrageMessageWrapper,groovy=groovy.lang.GroovyShell
+
+# Disabled by default, applications should enable this after their TLS certificates are
+# deployed and confirmed to work, as disabling this after the fact can result in clients
+# remembering the old value for a long time.
+http.add.header.Strict-Transport-Security.enabled = false
+http.add.header.Strict-Transport-Security.value = max-age=31536000; includeSubDomains
+
+# Disable by default, but can be enabled if iframe widgets will not be used from other origins.
+# This can optionally be fine tuned to allow specific origins or same origin loading.
```

Comment:
@niloc132 Is this true? I didn't think X-Frame-Options allowed specifying origins

Reply:
Sort of. https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Headers/X-Frame-Options says you can specify `ALLOW-FROM origin`, but also notes that this is obsolete, modern browsers will ignore it (which is good for us). If deployments set that for old browsers (that is the assumption given in the failing security audits...), then it will protect those users by not letting them access these resources. CSP, if it is enabled, should specify what domains they can use iframes from.

Reply:
I wonder what the "ignore" behavior is. Does it default to "DENY" or act as no header was set at all.

Reply:
from https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Headers/X-Frame-Options#allow-from_origin

> Modern browsers that encounter response headers with this [(ALLOW-FROM origins)] directive will ignore the header completely. The [Content-Security-Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Headers/Content-Security-Policy) HTTP header has a [frame-ancestors](https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Headers/Content-Security-Policy/frame-ancestors) directive which you should use instead.